tests/store/metadata/oxigraph/test_oxigraph_editions.py

```diff
@@ -0,0 +1,29 @@
+import os
+import pytest
+
+from pydantic import ValidationError
+
+from store.metadata.oxigraph.store import OxigraphMetadataStore
+import schemas
+
+
+def test_oxigraph_get_editions_returns_valid_structure():
+    """
+    Confirm that the OxigraphMetadataStore.get_edition()
+    function returns an edition that matches the edition
+    schema.
+    """
+
+    store = OxigraphMetadataStore()
+
+    editions = store.get_editions("cpih")
+    editions_schema = schemas.Editions(**editions)
+    assert editions_schema.id == "https://staging.idpd.uk/datasets/cpih/editions"
+    assert editions_schema.type == "hydra:Collection"
+    assert len(editions_schema.editions) == 1
```

Comment:
can we change this to assert len > 0 please. that way if we update that sample dataset with more editions later on we won't need to update the test.